backend/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Prerequisites
 
-- [GoLang 1.6](https://golang.org/doc/install)
+- [GoLang 1.16](https://golang.org/doc/install)
```

Comment:
i don't think this is needed I am running 1.18 and we have other ppl running 1.16.